dev-env/bin/ghc

```diff
@@ -0,0 +1 @@
+../lib/dade-exec-nix-tool
```

Comment:
ghcide expects this in PATH now

Reply:
Out of curiosity, do you know why `ghcide` now requires `ghc` in `PATH`?

Reply:
I think it’s `ghc-check` which is used to check for version mismatches.